lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -258,7 +261,7 @@ private JWEObject signEncryptJar(
             String criId)
             throws HttpResponseExceptionWithErrorBody, ParseException, JOSEException {
         SharedClaimsResponse sharedClaimsResponse =
-                getSharedAttributes(userId, currentVcStatuses, criId);
+                getSharedAttributes(ipvSessionItem, userId, currentVcStatuses, criId);
```

Comment:
can just pass IpvSessionItem.getEmail() instead of whole object.

Reply:
I think it would be more appropriate to pass the object to methods for clean code and need more than this properties in method.